tangelo/linq/tests/test_symbolic_simulator.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A test class to check that the simulator class functionalities are behaving
+as expected for the symbolic backend.
+"""
+
+import unittest
+
+from tangelo.linq import Gate, Circuit
+from tangelo.helpers.utils import installed_backends
+
+from tangelo.helpers.utils import installed_backends
+from tangelo.linq.target.target_sympy import SympySimulator
+
+
+class TestSymbolicSimulate(unittest.TestCase):
```

Comment:
After your changes, I think this format may be automatically added to our performance test.
It may be interesting to see how this does: I don't expect it to be fast or practical beyond examples of modest size. If that's the case the user should get a disclaimer: you designed this backend for specific reasons.

Reply:
This is indeed very slow, and it prevents the test workflow to be completed in < 2h. Would it be OK to remove the sympy translation from the performance tests?

Reply:
Yes, let's just use it on tests that make sense.